v2/googlecloud-to-googlecloud/src/test/java/com/google/cloud/teleport/v2/transforms/SpannerToBigQueryTransformTest.java

```diff
@@ -22,44 +22,37 @@
 import org.apache.beam.sdk.testing.TestPipeline;
 import org.apache.beam.sdk.transforms.Create;
 import org.apache.beam.sdk.values.PCollection;
-
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
-import java.util.Collections;
-import java.util.List;
-
-/** Unit test for {@link com.google.cloud.teleport.v2.transforms.SpannerToBigQueryTransform} methods. */
+/**
+ * Unit test for {@link com.google.cloud.teleport.v2.transforms.SpannerToBigQueryTransform} methods.
+ */
 @RunWith(JUnit4.class)
 public class SpannerToBigQueryTransformTest {
-  @Rule
-  public final transient TestPipeline pipeline = TestPipeline.create();
+  @Rule public final transient TestPipeline pipeline = TestPipeline.create();
 
   @Test
   public void testStructToJsonValidInput() {
     String expectedJson = "{\"bool\":true,\"string\":\"test-string\",\"int\":2,\"float\":3.0}";
-    Struct testStruct = Struct.newBuilder()
-        .set("bool")
-        .to(Value.bool(true))
-        .set("string")
-        .to(Value.string("test-string"))
-        .set("int")
-        .to(Value.int64(2))
-        .set("float")
-        .to(Value.float64(3.0))
-        .build();
-
-    List<Struct> inputStruct = Collections.singletonList(testStruct);
+    Struct testStruct =
```

Comment:
I think this would be cleaner if it were formatted as:

```
Struct.newBuilder()
    .set("bool").to(Value.bool(true))
    ...
    .build()
```

That'll require disabling spotless for the lines of code. There's an example here: https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/9569622f3f86653228425554cc6572da32f104fd/src/test/java/com/google/cloud/teleport/spanner/CopyDbTest.java#L93-L124

Reply:
Along with the spotless comments, you also should change this so that both `set` and `to` are on the same line. Disabling spotless is just intended to prevent it from being changed back when you run `mvn spotless:apply`.